### PR TITLE
server-server-api: Fix grammar

### PR DESCRIPTION
**Pull request by progval**
Preview: https://pr1665--matrix-spec-previews.netlify.app

### CODE REVIEW

File: content/server-server-api.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ to send. The process overall is as follows:
     for `<hostname>`.
 
 6.  If the `/.well-known` request returned an error response, and the
-    SRV record was not found, an IP address is resolved using CNAME, AAAA and A
+    SRV records were not found, an IP address is resolved using CNAME, AAAA and A
```
Comment (MTRNord) [inline content/server-server-api.md:190]: Isn't matrix set up in a way where there only is one SRV record at any given time? (Though on a quick search I didnt see anything not allowing multiple)

Comment (uhoreg) [reply content/server-server-api.md:190]: Standard SRV resolution allows for multiple records, and Matrix never forbade it.  But also, steps 4 and 5 refer to different SRV records (`_matrix-fed._tcp.<hostname>` and `_matrix._tcp.<hostname>`), so it really should be plural.

On the other hand "the SRV records were not found" sounds like SRV records exist but could not be found for whatever reason.  So maybe something like "... and no SRV records are present ...", or "... and no SRV records are found ..." might be better?

File: content/server-server-api.md [hunk 2]
```diff
@@ -187,7 +187,7 @@ to send. The process overall is as follows:
     for `<hostname>`.
 
 6.  If the `/.well-known` request returned an error response, and the
```
Comment (uhoreg) [inline content/server-server-api.md:189]:
```suggestion
6.  If the `/.well-known` request returned an error response, and
```

**Review by uhoreg — APPROVED** [review]
Thanks!

**Pull request merged**